code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -37,16 +45,16 @@
 	adjustFireLoss(0)
 
 /mob/living/silicon/robot/proc/use_power()
-	if(is_component_functioning("power cell") && cell)
+	if(cell && is_component_functioning("power cell"))
```

Comment:
Shouldn't it be get_cell() ?

Reply:
cell is already in its typecast, if robot's get_cell() did anything other than returning cell, maybe.